test/TestDeprecatedModule.py

```diff
@@ -1,10 +1,9 @@
 import unittest
+from test import ANSIBLE_MAJOR_VERSION, RunFromText
```

Comment:
isort should be made aware of the first-party test helpers. It's best to use an explicit relative import for this.